scalableminds/webknossos
# Enforce minimum-read-time for new task description modal

This avoids that a new task description is dismissed by accident. A minimum on ten seconds should be invested in understanding the new task type.

Also this PR uses the markdown renderer for the task type description.

### Mailable description of changes:
 - When getting a new task, the new description is shown in a modal which can only be closed after ten seconds. These ten seconds should be used to fully understand the new task type.

### URL of deployed dev instance (used for testing):
- https://minimumtimeonchanged-description.webknossos.xyz

### Steps to test:
- create two task types A, B with different descriptions
- create a task for A and a task for B
- get the first task, trace and click "get next task" in the tracing window
The following modal should show:
![image](https://user-images.githubusercontent.com/2486553/41782423-79797b56-763a-11e8-92e6-07cd563c295b.png)

The modal should only be closable after ten seconds.

### Issues:
- fixes #2791 and #2792 

------
